homeassistant/helpers/translation.py

```diff
@@ -197,23 +197,25 @@ def __init__(self, hass: HomeAssistant) -> None:
         """Initialize the cache."""
         self.hass = hass
         self.loaded: dict[str, set[str]] = {}
-        self.cache: dict[str, dict[str, dict[str, Any]]] = {}
+        self.cache: dict[str, dict[str, dict[str, dict[str, str]]]] = {}
 
     async def async_fetch(
         self,
         language: str,
         category: str,
         components: set[str],
```

Comment:
TLDR: the set is not at the end of the cache instead of having to transverse it in the middle of the cache